packages/credential-w3c/src/action-handler.ts

```diff
@@ -16,6 +16,8 @@ import {
   W3CVerifiablePresentation,
 } from '@veramo/core'
 
+import { CredentialStatus } from 'credential-status'
```

Comment:
This import should be paired with a declaration of `credential-status` in `package.json` OR we can remove the import entirely and use a local type instead.

This is a very simple data type that can be re-declared internally here, or simply used as `any` type.